suricata/update/main.py

```diff
@@ -88,7 +88,7 @@
     logger = logging.getLogger()
 
 # If Suricata is not found, default to this version.
-DEFAULT_SURICATA_VERSION = "4.0.0"
```

Comment:
Should this deserve an upgrading note somewhere, considering that ET, for instance, has rulesets for older versions, so this might affect some of their folks? 🤔

Reply:
I think this is fine, we don't have an upgrade section and its easy to work-around this.